boilerplate/flyte/golang_test_targets/Makefile

```diff
@@ -16,11 +16,9 @@ generate: download_tooling #generate go code
 lint: download_tooling #lints the package for common code smells
 	GL_DEBUG=linters_output,env golangci-lint run --deadline=5m --exclude deprecated -v
 
-# If code is failing goimports linter, this will fix.
-# skips 'vendor'
-.PHONY: goimports
-goimports:
-	@boilerplate/flyte/golang_test_targets/goimports
+.PHONY: lint-fix
+lint-fix: download_tooling #lint the package for common code smells. fixes files in-place
+	GL_DEBUG=linters_output,env golangci-lint run --fix --deadline=5m --exclude deprecated -v
```

Comment:
How about we combine these two make targets?

```
LINT_FLAGS ?=

.PHONY: lint
lint: download_tooling
	GL_DEBUG=linters_output,env golangci-lint run $(LINT_FLAGS) --deadline=5m --exclude deprecated -v

.PHONY: lint-fix
lint-fix: LINT_FLAGS=--fix
lint-fix: lint

```

Reply:
That's a good suggestion so these are never out of sync, will push a fix in a moment.